[0.5.0+ regression] type error with new releases

v0.5.0 and v0.5.1 have a typescript type error.

To reproduce:
```
mkdir langsmith-type-issue
cd langsmith-type-issue
npm init -y
npm pkg set type=module
npm i typescript langsmith@0.5.0
npx tsc --init --skipLibCheck false --exactOptionalPropertyTypes false
echo "import * as langsmith from 'langsmith'; console.log(langsmith);" > main.ts
npx tsc --noEmit
```

results in:
```
node_modules/langsmith/dist/index.d.ts:7:88 - error TS2305: Module '"./utils/prompt_cache/index.js"' has no exported member 'promptCacheSingleton'.

7 export { PromptCache, type CacheConfig, type CacheMetrics, configureGlobalPromptCache, promptCacheSingleton, } from "./utils/prompt_cache/index.js";
                                                                                         ~~~~~~~~~~~~~~~~~~~~

Found 1 error in node_modules/langsmith/dist/index.d.ts:7
```

--- a/js/src/utils/prompt_cache/index.ts
+++ b/js/src/utils/prompt_cache/index.ts
@@ -335,7 +335,6 @@ export class PromptCache {
 }
 
 /**
- * @internal
  * Global singleton instance of PromptCache.
  * Use configureGlobalPromptCache(), enableGlobalPromptCache(), or disableGlobalPromptCache() instead.
  */

